tests/channels/pjsip/info_dtmf/sipp/dtmf-duration.xml

```diff
@@ -272,7 +272,7 @@
       Content-Type: application/dtmf-relay
       Content-Length: [len]
 
-      Signal=10
+      Signal=A
```

Comment:
Were these incorrect in the first place?

Reply:
No.  Either is acceptable but the letters/symbols are clearer than the numbers.  I couldn't find the numbers for the symbols anyway. :)